jbmc/src/java_bytecode/assignments_from_json.cpp

```diff
@@ -902,11 +902,10 @@ code_with_references_listt assign_from_json_rec(
   return result;
 }
 
-void assign_from_json(
+code_with_references_listt assign_from_json(
```

Comment:
:pick: Typo in the commit message: `the all the value pairs` -> `all the value pairs`